Source/Core/InputCommon/QuartzInputMouse.h

```diff
@@ -0,0 +1,24 @@
+#include <ApplicationServices/ApplicationServices.h>
+#include "GenericMouse.h"
+
+namespace prime
+{
+
+bool InitCocoaInputMouse(uint32_t* windowid);
+CGRect getBounds(uint32_t* m_windowid);
+CGPoint getWindowCenter(uint32_t* m_windowid);
```

Comment:
`getBounds` and `getWindowCenter` don't need to be exposed outside of `QuartzInputMouse.mm`, so they shouldn't be declared here. Mark them as static.